packages/cardano-services/package.json

```diff
@@ -61,7 +65,6 @@
     "dockerode": "^3.3.1",
     "dockerode-utils": "^0.0.7",
     "get-port-please": "^2.5.0",
-    "npm-run-all": "^4.1.5",
```

Comment:
:thinking: Usually [locally referenced](https://github.com/input-output-hk/cardano-js-sdk/pull/312/commits/5764345374eb2300a35f7ebb9252fc665cb6ec69#diff-718fd329469aa32e693b48626b09b792498085607dae1c0f8404fe5a8970d848L28) devDeps need to be defined in the local `package.json`, as the executables are unpacked locally.

Reply:
This now works due to emulation: https://github.com/input-output-hk/cardano-js-sdk/pull/302/files#diff-206b9ce276ab5971a2489d75eb1b12999d4bf3843b7988cbe8d687cfde61dea0R28

Included into the build dependencies: https://github.com/input-output-hk/cardano-js-sdk/pull/302/files#diff-206b9ce276ab5971a2489d75eb1b12999d4bf3843b7988cbe8d687cfde61dea0R38

Reply:
Sure, but we still need it defined here (and any other `package.json` that uses it)